tests/playwright/src/specs/volume-smoke.spec.ts

```diff
@@ -77,16 +83,102 @@ describe('Volume workflow verification', async () => {
     await volumeDetails.closeButton.click();
     await playExpect(volumesPage.heading).toBeVisible();
   });
+
+  test('Delete volume from Volumes page', async () => {
+    let volumesPage = await navBar.openVolumes();
+    await playExpect(volumesPage.heading).toBeVisible();
+    let volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    playExpect(volumeRow).not.toBeUndefined();
+    volumesPage = await volumesPage.deleteVolume(volumeName);
+    await playExpect.poll(async () => await volumesPage.waitForVolumeDelete(volumeName)).toBeTruthy();
+    volumeRow = await volumesPage.getVolumeRowByName(volumeName);
```

Comment:
I'm not sure I understand what this line does, why are you getting the volumeRow after you already deleted the volume, shouldnt it just be undefined here?